cucumber/authenticators_jwt/features/support/jwt_jwks_helper.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+# Utility methods for JWT and JWKs manipulation
+
+require 'openssl'
+require 'jwt'
+
+module JwtJwksHelper
+
+  module Algorithms
+    RS256 = "RS256".freeze
+  end
+
+  JWKS_ROOT_PATH = "/var/jwks".freeze
+  JWKS_BASE_URI = "http://jwks".freeze
+  BITS_2048 = 2048
```

Comment:
Add freezes on this consts too
Move const to another file or at least about the Algorthems Module